### DISCUSSION TITLE
Domain Name not working

### DISCUSSION THREAD

**Discussion by Alex-Giaquinto** [gravitational/teleport#38934] [General]
So I have been using teleport for roughly 4 months. Been working great and we rolled out teleport connect to the rest of our team. Out of no where, it just stopped working. I went to the server to debug and found literally nothing. I can still access it via the web browser but only by the IP address. The original domain name does not work now and I am lost. Has this happened to anyone before or can someone tell me where to look for a potential error?

**Comment by stevenGravy** [comment]
This sounds like a DNS issue. I would confirm `dig hostname` resolves to the same IP. Teleport only maintains DNSes for its hosted service.
